src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -472,87 +472,87 @@
             </trans-unit>
             <trans-unit id="122">
                 <source>This file is not a valid video.</source>
-                <target state="needs-review-translation">Diese Datei ist kein gültiges Video.</target>
+                <target>Diese Datei ist kein gültiges Video.</target>
             </trans-unit>
             <trans-unit id="123">
                 <source>The size of the video could not be detected.</source>
-                <target state="needs-review-translation">Die Größe des Videos konnte nicht ermittelt werden.</target>
+                <target>Die Größe des Videos konnte nicht ermittelt werden.</target>
```

Comment:
`Größe` is ambiguous here. Do we talk about the size in Bytes? Or the dimensions (width and height) of the video? Looking at the other translated strings, it's probably the latter.

Reply:
dimensions indeed

Reply:
Reading the code from #59042 this is indeed about the dimensions of the video. Let's fix that.